docs/contributing/coding-guidelines/ros-nodes/coordinate-system.md

```diff
@@ -1,5 +1,49 @@
 # Coordinate system
 
-!!! warning
+## Overview
 
-    Under Construction
+The commonly used coordinate systems include the world coordinate system, the vehicle coordinate system, and the sensor coordinate system.
+
+- The world coordinate system is a fixed coordinate system that defines the physical space in the environment where the vehicle is located.
+- The vehicle coordinate system is the vehicle's own coordinate system, which defines the vehicle's position and orientation in the world coordinate system.
+- The sensor coordinate system is the sensor's own coordinate system, which is used to define the sensor's position and orientation in the vehicle's coordinate system.
+
+## How coordinates are used in Autoware
+
+In Autoware, coordinate systems are typically used to represent the position and movement of vehicles and obstacles in space. Coordinate systems are commonly used for path planning, perception and control, can help the vehicle decide how to avoid obstacles and to plan a safe and efficient path of travel.
+
+1. Transformation of sensor data
+
+   In Autoware, each sensor has a unique coordinate system and their data is expressed in terms of the coordinates. In order to correlate the independent datas between different sensors, we need to find the position relationship between each sensor and the vehicle body. Once the installation position of the sensor on the vehicle body is determined, it will remain fixed during running, so the offline calibration method can be used to determine the precise position of each sensor relative to the vehicle body.
+
+2. ROS TF2
+
+   The `TF2` system maintains a tree of coordinate transformations to represent the relationships between different coordinate systems. Each coordinate system is given a unique name and they are connected by coordinate transformations. How to use `TF2`, refer to the [TF2 tutorial](http://docs.ros.org/en/galactic/Concepts/About-Tf2.html).
+
+## TF tree
+
+In Autoware, a common coordinate system structure is shown below:
+
+![tf-tree](./images/coordinate-system-tf-tree.png)
```

Comment:
```suggestion
```mermaid
graph TD
    /earth --> /map
    /map --> /base_link
    /base_link --> /imu
    /base_link --> /lidar
    /base_link --> /camera_link
    /camera_link --> /camera_optical_link```
```

You can use mermaid diagrams like here, they are light and better fit to documents.

https://mermaid.live/edit#pako:eNptj7sKwzAMRX_FaE7I7qFT-gXtVAxFtZXG1C8ceygh_163eZBCNEnnXAk0gvSKgMMzYujZtRWOlWoIY-pZXZ9YYzEssHQzeuBAd6PdaxHbPGtt87EwWmE8VhItRdwf3ZG_iA9JSzQ_ARUUZlGr8sL4XRSQerIkgJdWUYfZJAHCTSWKOfnL20ngKWaqIAeFiVqN5XkLvEMzbPSsdPJxTQZ0N-_XzPQBjaJk0Q